src/Microsoft.Diagnostics.Monitoring.WebApi/Controllers/DiagController.cs

```diff
@@ -75,10 +75,17 @@ public Task<ActionResult<IEnumerable<Models.ProcessIdentifier>>> GetProcesses()
                 {
                     defaultProcessInfo = await _diagnosticServices.GetProcessAsync(null, HttpContext.RequestAborted);
                 }
-                catch (Exception)
+                catch (ArgumentException)
                 {
                     // Unable to locate a default process; no action required
                 }
+                catch (InvalidOperationException)
+                {
+                }
+                catch (Exception ex) when (!(ex is OperationCanceledException))
+                {
+                    _logger.LogWarning(ex.Message);
```

Comment:
Please move to LoggingExtensions

Reply:
How about "Failed to determine the default process." and pass the exception in the exception parameter?